src/gitops/components/rollout/RolloutNavPage.tsx

```
@@ -34,7 +35,10 @@ const RolloutNavPage: React.FC<RolloutPageProps> = ({ name, namespace, kind }) =
     namespace,
   });
 
-  const [actions] = useRolloutActionsProvider(rollout);
+  const isRevisionsTab = /\/revisions\/?$/.test(location.pathname);
+  const [actions] = isRevisionsTab
+    ? useRolloutRevisionsActionsProvider(rollout)
```

ESLint

[error] React Hook "useRolloutRevisionsActionsProvider" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)